api/loader/fileloader_test.go

```diff
@@ -201,6 +201,30 @@ func TestLoaderBadRelative(t *testing.T) {
 	require.Error(err)
 }
 
+func TestLoaderAbsolute(t *testing.T) {
+	require := require.New(t)
+
+	l1, err := makeLoader().New("foo/project")
+	require.NoError(err)
+	require.Equal("/foo/project", l1.Root())
+
+	// It's not okay to escape out of the root directory
+	_, err = l1.New("/tmp/escapedpath")
+	require.Error(err)
+
+	// It's okay to cd into an absolute path
+	l2, err := l1.New("/foo/project/subdir2")
```

Comment:
I think this one should be probably: `l2, err := l1.New("subdir2")`

Reply:
We're testing the absolute path here, so I think this is correct.